fd/diff.go

```diff
@@ -45,8 +49,7 @@ type Settings struct {
 // of the function.
 func DefaultSettings() *Settings {
```

Comment:
Yea, we hadn't adopted either the "nil is okay" or "zero is reasonable" approach when this first went in. This can be removed.